test/sub_test.go

```diff
@@ -908,10 +918,19 @@ func TestChanSubscriberPendingLimits(t *testing.T) {
 			switch typeSubs {
 			case 0:
 				sub, err = nc.ChanSubscribe("foo", ch)
+				if err := sub.SetPendingLimits(pending, -1); err == nil {
```

Comment:
I thought limits did not apply to Channel subscribe, because the limit is really whatever the user sets for the provided channel?

Reply:
The test expects an error since its checking for `err == nil` but the t.Fatalf descriptions are wrong, will fix.